src/Analysis/GWASResults/Views/Home/HomeTable/ActionsDropdown/ActionsDropdown.jsx

```diff
@@ -31,6 +34,28 @@ const ActionsDropdown = ({ record }) => {
       ),
       disabled: false,
     },
+    {
+      key: '2',
+      label: (
+        <a
+          href=''
+          onClick={(e) => {
+            e.preventDefault();
+            retryWorkflow(
+              record.name,
+              record.uid,
+            ).then(() => {
+              alert('Workflow successfully restarted.');
+            }).catch((error) => {
+              alert(`Retry request failed. \n\n${error}`);
+            });
+          }}
+        >
+          Retry
+        </a>
+      ),
+      disabled: (!(record.phase === PHASES.Error || record.phase === PHASES.Failed)),
```

Comment:
I personally find this easier to read

```
      disabled: record.phase !== PHASES.Error && record.phase !== PHASES.Failed,
```

Reply:
solved by adding `// eslint-disable-line` to this line.